### PR TITLE
Add initial documentation about role usage

### PR DESCRIPTION
**Pull request by tadeboro**
This is a first attempt at documenting the use cases that collection covers. We started with a sandbox scenario because this is what most beginners will likely need when setting up a Sensu Go environment for testing.

### CODE REVIEW

File: docs/source/sandbox.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+Settign up a demo environment
```
Comment (matejart) [inline docs/source/sandbox.rst]:
```suggestion
Setting up a demo environment
```

File: docs/source/sandbox.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+=============================
+
+If we would like to set up a simple Sensu Go sandbox, we just need a following
```
Comment (matejart) [inline docs/source/sandbox.rst]:
```suggestion
To set up a simple Sensu Go sandbox, we just need the following
```

File: docs/source/sandbox.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+     children:
+       backends:
+         hosts:
+           192.168.50.4:
+
+       agents:
+         hosts:
+           192.168.50.5:
+
+You can safely ignore the ``ansible_ssh_common_args`` if can reach your hosts
```
Comment (matejart) [inline docs/source/sandbox.rst]:
```suggestion
You can safely ignore the ``ansible_ssh_common_args`` if you can reach your hosts
```

File: docs/source/sandbox.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+via ssh. We just left it there since this snippet comes in handy sometimes.
+
+All that we need to do now is run the playbook::
```
Comment (matejart) [inline docs/source/sandbox.rst]:
```suggestion
All that we need to do now is to run the playbook::
```

File: docs/source/sandbox.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+         include_role:
+           name: sensu.sensu_go.agent
+         vars:
+           agent_config:
+             # Place your configuration here
+
+When we run it, it will install and configure backend and agents on selected
+hosts. Note that we do not need to inform agents explicitly where the backend
+is, because the role can deduce this on its own if the backend host group is
+named ``backends``.
```
[the remark this answers is not in the document]
Comment (tadeboro) [reply docs/source/sandbox.rst]: This is the result of missing a `backends` group:

```
fatal: [192.168.50.5]: FAILED! => {"changed": false, "msg": "AnsibleError: Unexpected templating type error occurred on (---\n#\n# {{ managed }}\n#\n\n##\n# Sensu agent configuration\n##\n{# -------------------------------------- #}\nbackend-url:\n  {% for url in agent_backend_urls %}\n  - {{ url }}\n  {% endfor %}\n{# -------------------------------------- #}\n{% if agent_log_level %}\nlog-level: {{ agent_log_level }}\n{% endif %}\n{# -------------------------------------- #}\n): 'NoneType' object is not iterable"}
```

**Comment by matejart** [comment]
Looks good, thank you! 👍

**Pull request merged**